content/docs/1.0.0/tutorial/manage-your-microservices.md

```diff
@@ -2,79 +2,97 @@
 
 In this step of the tutorial we're ready to learn the basics of managing microservices inside our [Tutorial project](../tutorial/introduction.html).
 
-## Basic Commands
+## Basic Controller CLI Interactions
 
 The Agent daemon runs microservices on your edge nodes locally, but it is controlled remotely by the Controller. Let's learn some of the most common Controller commands you'll use.
 
-To start, let's enter a shell into our locally running iofog-controller container:
-
-```sh
-docker exec -ti iofog-controller bash
-```
-
 <aside class="notifications note">
   <h3><img src="/images/icos/ico-note.svg" alt=""> Production vs. Development</h3>
   <p>Throughout the rest of this tutorial remember that, while we're running our ioFog network entirely locally using Docker, in production it doesn't have to be--and probably won't. Your Controller and Connector will likely be running on cloud servers and your edge node devices in the wild will each be running their own Agent.</p>
 </aside>
 
-This tutorial includes 5 microservices already running. We can view any configured microservices with `microservice list`:
+This tutorial includes 3 microservices already running. We can view any configured microservices using Controller CLI: `microservice list`.
 
-```sh
-iofog-controller microservice list
+```bash
+docker exec -ti iofog-controller iofog-controller microservice list
 ```
 
 This returns a JSON object containing the full list, along with their configuration. The most important microservices currently running are:
 
-###### Sensors
+###### Tutorial Microservices
 
-A microservice that pretends to be reading data from a local hardware sensor. The data it produces is published with [the SDK](../writing-microservices/sdk.html) and routed through the [Connector](../connectors/overview.html) so that it can be read by other microservices.
+The tutorial consists of three microservices deployed on top of ioFog stack.
```

Comment:
three > 3

(Normally though, for single digit numbers, you use the wording but we're not being finicky because this is not an actual paper haha. The rule of thumb is usually to stick with the same format as much as possible and definitely use numerals for two-digit numbers.)